lib/services/packagist.rb

```diff
@@ -37,14 +36,13 @@ def domain
 
   def full_domain
     if data['domain'].to_s == ''
-      'http://packagist.org'
+      'https://packagist.org'
     else
       data['domain']
-    end.lstrip.sub(/[\/\s]+$/,'')
+    end.lstrip.sub(/[\/\s]+$/,'').sub(/^http:\/\/packagist.org/, 'https://packagist.org')
```

Comment:
That's only true with the `m` modifier though? Unless ruby has that on by default, I don't know it much :)